src/modules/droplet/delete-droplet-by-tag/delete-droplet-by-tag.ts

```diff
@@ -0,0 +1,18 @@
+import { IResponse, IContext } from '../../../types';
+
+export interface IDeleteDropletByTagApiRequest {
+  tag_name: string;
+}
+
+export type DeleteDropletByTagResponse = IResponse<void>;
+
+export const deleteDropletByTag = ({
+  httpClient,
+}: IContext) => ({
+  tag_name,
+}: IDeleteDropletByTagApiRequest): Promise<Readonly<DeleteDropletByTagResponse>> => {
+    const path = '/droplets';
+    const url = `${path}?tag_name=${tag_name}`;
+
+    return httpClient.delete(url);
```

Comment:
I think here we could pass the query parameter via the "params", 
before merge I will change it ;)